cmd/podman/create.go

```diff
@@ -663,7 +671,17 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 			}
 		}
 	}
+	// This is done because cobra cannot have two aliased flags. So we have to check
+	// both
```

Comment:
Can we do a check if net and network are both set and error if so?

Reply:
im going to have to go through some of this at another time.  i will add that as a good idea.